packages/FormElement/stories/examples/ButtonGroupExample.js

```diff
@@ -14,7 +15,11 @@ export default function ButtonGroupExample() {
   return (
     <FormElement label="Form Label" onClickLabel={() => refButtonGroup.current.focus()}>
       <FormElement.Content>
-        {() => <ButtonGroup ref={refButtonGroup}>{buttonGroupOptions}</ButtonGroup>}
+        {() => (
+          <ButtonGroup style={{ marginTop: stylers.spacer(2) }} ref={refButtonGroup}>
```

Comment:
the consumers can set the increased margin for the content if needed. (Using css or inline style)